endpoints-support.md

```diff
@@ -0,0 +1,581 @@
+|<sub>/rest/backups/remotearchive/{id}</sub>                                              |DELETE    |    |    |
+|<sub>/rest/backups/{id}</sub>                                                            |GET       |    |    |
+|<sub>/rest/backups/{id}</sub>                                                            |DELETE    |    |    |
+|     **Certificate Authority**                                                                                                                    |
+|<sub>/rest/certificates/ca</sub>                                                         |GET       |    |    |
+|<sub>/rest/certificates/ca/crl</sub>                                                     |GET       |    |    |
+|<sub>/rest/certificates/ca/{aliasName}</sub>                                             |DELETE    |    |    |
+|     **Certificates Client RabbitMq**                                                                                                             |
+|<sub>/rest/certificates/client/rabbitmq</sub>                                            |POST      |    |    |
+|<sub>/rest/certificates/client/rabbitmq/keypair/{aliasName}</sub>                        |GET       |    |    |
+|<sub>/rest/certificates/client/rabbitmq/keys/{aliasName}</sub>                           |GET       |    |    |
+|<sub>/rest/certificates/client/rabbitmq/{aliasName}</sub>                                |GET       |    |    |
+|     **Connection Template**                                                                                                                      |
+|<sub>/rest/connection-templates</sub>                                                    |GET       | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/connection-templates/defaultConnectionTemplate</sub>                          |GET       | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/connection-templates/{id}</sub>                                               |GET       | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/connection-templates/{id}</sub>                                               |PUT       | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|     **Datacenter**                                                                                                                               |
+|<sub>/rest/datacenters</sub>                                                             | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/datacenters</sub>                                                             | POST     | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/datacenters</sub>                                                             | DELETE   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
```

Comment:
I did not found this endpoint for DELETE method.
The delete for a set of resources was not implemented to our resources of ruby sdk, just the delete of one resource by id =/

Reply:
Indeed.

For someone using the SDK however, this basically boils down to something like:
```ruby
datacenter_class.multi_remove(@client, width: 11000)
```
instead of:
```ruby
datacenter_class.find_by(@client, width: 11000).map(&:remove)
```

In the case of the REST API it does need to be separated since you can only make one request at a time, but for the SDK the *functionality* of removing multiple datacenters itself is already in there, since the user has access to *loops*.

An argument *could* definitely be made in regards to the number of requests to the appliance, which would be 1 for that endpoint vs `X` using `&:remove`, `X` being the number of resources found.

But, since we usually don't use filters such as `filter="name matches 'XYZ%25'` in the SDK and tend to use `find_by` even internally, the implemented method would probably behave similarly to what I described.

@tmiotto @ricardogpsf @aalexmonteiro  , do share your thoughts on this.

Reply:
I agree with that approach, it is more simple and the user get the same result.
If need, we can document that, because the user cannot call just one method to apply the desired action, he need iterate "manually" for apply it.
If not need document that, no problem for me. =) 
We did this approach in the integration tests. 😀 😆

Reply:
Sounds good to me, since it is a simple solution.